lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -158,26 +160,28 @@ def test_query(self, client, server_config, query_helper, user_ok, find_template
         index = self.build_index(server_config, ("2020-08", "2020-09", "2020-10"))
         response = query_helper(json, index, 200, server_config, json=response_payload)
         res_json = response.json
-        assert isinstance(res_json, list)
-        assert len(res_json) == 1
-        assert (
-            res_json[0]["key"]
-            == "fio_rhel8_kvm_perf43_preallocfull_nvme_run4_iothread_isolcpus_2020.04.29T12.49.13"
-        )
-        assert res_json[0]["@metadata.controller_dir"] == "dhcp31-187.example.com"
+        assert isinstance(res_json, dict)
+        assert len(res_json.keys()) == 1
+        data = res_json["dhcp31-187.example.com"]
+        assert isinstance(data, list)
+        assert len(data) == 1
+        # assert (
+        #     data[0]["key"]
+        #     == "fio_rhel8_kvm_perf43_preallocfull_nvme_run4_iothread_isolcpus_2020.04.29T12.49.13"
+        # )
```

Comment:
Should this code be removed?

Reply:
Yeah; I glanced through quickly right after I posted, and noticed this as well. Again, I was on the edge about removing `"key"` so I guess you could say I kept half of it. Maybe 1/4 since it's only commented unit test code ... ;-)